libs/common/src/config/app_info_builder.cpp

```diff
@@ -21,9 +21,9 @@ tl::expected<std::string, Error> AppInfoBuilder::Tag::Build() const {
 }
 
 bool ValidChar(char c) {
-    if(c > 0 && c < 255) {
```

Comment:
Could put a `static_assert(std::numeric_limits<char>::max() <= 0xFF)`. Is that too paranoid?